test_scripts/upgrade.sh

```diff
@@ -20,104 +27,136 @@ echo "Checking out $START_BRANCH branch"
 git checkout $START_BRANCH
 
 ## Change to molecule directory on pre 7.0 branches
-
-cd roles/confluent.test/
+if [[ $START_BRANCH != 7* ]]
```

Comment:
What if version is 8.0

Reply:
Thanks Neeraj. Will make changes across to make it workable with 8.x.y and further branch.